app/javascript/src/components/Common/PrivateRoute.jsx

```diff
@@ -1,25 +1,35 @@
 import React from "react";
 import { Redirect, Route } from "react-router-dom";
 import PropTypes from "prop-types";
+import { either, isEmpty, isNil } from "ramda";
+
+import Hero from "components/Home/Hero";
 
 const PrivateRoute = ({
   component: Component,
   condition,
   path,
-  redirectRoute,
+  redirectRoute = null,
   ...props
 }) => {
-  if (!condition) {
+  if (condition) {
+    return <Route path={path} component={Component} {...props} />;
+  }
+
+  if (either(isNil, isEmpty)(redirectRoute)) {
+    return <Hero />;
+  } else {
```

Comment:
If user isn't authenticated redirect them to Hero page. Handle that in the main route file. PrivateRoute is a helper component. Custom logic shouldn't be handled in it. Please revert the PrivateRoute file.